specification/ai/Azure.AI.Unified/agents/common/models.tsp

```diff
@@ -0,0 +1,176 @@
+import "@typespec/http";
+import "@typespec/versioning";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace Azure.AI.Unified.Agents;
+
+@doc("The possible values for roles attributed to messages in a thread.")
+union MessageRole {
+  string,
+
+  @doc("The role representing the end-user.")
+  user: "user",
+
+  @doc("The role representing the agent.")
+  agent: "assistant",
+}
+
+@doc("The available sorting options when requesting a list of response objects.")
+union ListSortOrder {
+  string,
```

Comment:
What other sort orders will we ever support? This seem to be one of those instances where an open set of strings is not needed?